core/src/main/scala/geotrellis/raster/ArrowTensor.scala

```diff
@@ -88,6 +89,21 @@ case class ArrowTensor(val vector: Float8Vector, val shape: Seq[Int]) extends Ce
     ArrowTensor(result, shape)
   }
 
+  def getPixelVector(col: Int, row: Int): Vector = {
+    val pixelStack =
+      for (depth <- 0 until shape(0)) yield {
+    //    println("handling depth: ", depth)
+        val i = depth * rows * cols + row * cols + col
```

Comment:
Is this stride correct? Perhaps we should preserve it as a utility function or the like?

Reply:
@jpolchlo I'm curious if this looks correct to you: depth, then rows, then cols